packages/extension/src/inpage/alephiumWindowObject.ts

```diff
@@ -116,12 +116,6 @@ export const alephiumWindowObject: AlephiumWindowObject =
         throw Error("User aborted")
       }
 
-      const { alephiumProviders } = window
```

Comment:
Since `window.alephiumProviders.alephium` may be `undefined`, and we will detect injected providers in the dApp, we need to remove the code.